src/api-doc/Entity Management API's.postman_collection.json

```diff
@@ -594,15 +714,15 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{ \"query\": {\n        \"_id\": {\n            \"$in\": [\"5fd1b52ab53a6416aaeefb1f\", \"5fd1b52ab53a6416aaeefb24\",\"5f33c3d85f637784791cd830\",\"5fd1b52ab53a6416aaeefb26\",\"5fd098e2e049735a86b748ae\"]\n        }\n    },\n    \"projection\": [\n        \"_id\",\"metaInformation.name\"\n    ]}",
+							"raw": "{\n    \"query\": {\n        \"_id\": {\n            \"$in\": [\n                \"5fd1b52ab53a6416aaeefb1f\",\n                \"5fd1b52ab53a6416aaeefb24\",\n                \"5f33c3d85f637784791cd830\",\n                \"5fd1b52ab53a6416aaeefb26\",\n                \"5fd098e2e049735a86b748ae\"\n            ]\n        }\n    },\n    \"projection\": [\n        \"_id\",\n        \"metaInformation.name\"\n    ]\n}",
 							"options": {
 								"raw": {
 									"language": "json"
 								}
 							}
 						},
 						"url": {
-							"raw": "{{baseUrl}}/entity-management/v1/entities/find",
```

Comment:
@Prajwal17Tunerlabs in the url I think /entity-management can be part of base url. because we use admin instance of entity service as well in that case "entity-management" become "admin-entity"